prov/util/src/util_shm.c

```diff
@@ -43,22 +43,29 @@
 #include <fi_shm.h>
 
 
+static void smr_peer_addr_init(struct smr_addr *peer)
+{
+	memset(peer->name, '\0', SMR_NAME_SIZE);
```

Comment:
nit: memset wants an int as the 2nd arg, not a char.  So just 0 works better here.